cameraview/src/main/java/com/otaliastudios/cameraview/video/SnapshotVideoRecorder.java

```diff
@@ -141,7 +143,20 @@ public void onRendererFrame(@NonNull SurfaceTexture surfaceTexture, int rotation
                 case H_264: videoType = "video/avc"; break; // MediaFormat.MIMETYPE_VIDEO_AVC:
                 case DEVICE_DEFAULT: videoType = "video/avc"; break;
             }
-            String audioType = "audio/mp4a-latm";
+            String audioType = "";
+            if (mResult.audioCodec == AudioCodec.AMR_NB) {
+                audioType = "audio/3gpp";
+            } else if (mResult.audioCodec == AudioCodec.AMR_WB) {
+                audioType = "audio/amr-wb";
+            } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
+                    && mResult.audioCodec == AudioCodec.VORBIS) {
+                audioType = "audio/vorbis";
+            } else if (mResult.audioCodec == AudioCodec.AAC
+                    || mResult.audioCodec == AudioCodec.HE_AAC
+                    || mResult.audioCodec == AudioCodec.AAC_ELD
+                    || mResult.audioCodec == AudioCodec.DEVICE_DEFAULT) {
+                audioType = "audio/mp4a-latm";
+            }
```

Comment:
Can we do this with a `switch` like video does? So that if we add new codecs later, we'll get a compile error here.